src/inc/holder.h

```diff
@@ -62,6 +61,13 @@
         _NAME & operator=(_NAME const &);
 #endif
 
+// The types of stack validation we support in holders.
```

Comment:
Can be cleaned up later

Reply:
I fixed the other one, but this one I would prefer to do separately.